src/core/symbology/qgsrendererregistry.cpp

```diff
@@ -95,6 +96,10 @@ QgsRendererRegistry::QgsRendererRegistry()
                                         QIcon(),
                                         nullptr,
                                         QgsRendererAbstractMetadata::PolygonLayer ) );
+
+  addRenderer( new QgsRendererMetadata( QStringLiteral( "embeddedSymbol" ),
```

Comment:
TODO: icon? I don't think it should block merging of this PR.

Reply:
Yep - I've no inspiration for this particular one. I've created a new "3.20 icon updates" project for tracking new icons we need to focus on for 3.20: https://github.com/qgis/QGIS/projects/1